apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -244,7 +256,7 @@ export default function ListBox({
   return (
     <StyledInfiniteLoader
       isItemLoaded={isItemLoaded}
-      itemCount={count}
```

Comment:
the maximum amount of data that can be rendered is 200. In a very extreme case (very very big screen), the dom wants to render more than 200 data, and the rest will be empty which is a known limitation.